data_steward/spec/pages/file_transfer_procedures.md

```diff
@@ -1,43 +1,98 @@
- 
-     * __Subject__ New User Request: Data Steward
-     * Attestation of having completed your institutional security training
-     * Attestation of having entered your information in the [Data Steward Contact List](https://docs.google.com/spreadsheets/d/1Slh4teXKBwtD_ZrTFEjUTVH3Jk5ii-tcSJxocZnAqdo/edit?usp=sharing)
-     * Attached a signed, scanned copy of the [Rules of Behavior for Privileged Use](https://docs.google.com/document/d/1E6bRJ4l7AclEkaFS4Tg2zt9u3WyFMOpu4-omMjhlTRM/edit?usp=sharing). Note that the entire document must be attached, not just the signed page 
+---
+## KEYWORDS
+   * __SOP:__ Standard Operating Procedure
+   * __DRC:__ Data and Research Center
+   * __EHR:__ Electronic Health Records
+   * __Bucket:__ Location within Google Cloud Environment a sites data is stored
+   * __Data Steward:__ Individual responsible for uploading EHR data
+---
+## RESPONSIBILITY
+Each Organization tasked with uploading EHR data files or other Participant data to the DRC is required to request creation of a Bucket and register personnel who will fulfil the Data Steward role. The Data Steward is an individual at an Organization responsible for uploading EHR data to their Organization’s specified bucket within the DRC. An Organization may designate two (2) Data Stewards to provide backup coverage.  The responsibility of de-provisioning access is each Organization’s.  De-Provisioning or the unregistration of personnel designated as Data Steward or of an Organization itself is completed by sending an email to sysadmin@pmi-ops.org.  It is the organization's designated Data Steward(s) which is/are responsible for adherence to this procedure.
 
-1. Within five business days you should receive a response with:
-     * Instructions on how to complete registration of your `pmi-ops.org` account
-     * The name of your site's assigned bucket on Google Cloud Storage
-     * A test data set to be used during alpha
-     * Information on how to activate your account on our [JIRA site](https://precisionmedicineinitiative.atlassian.net) where you can report and track issues related to this process
-  
-      Keep this information in a safe place and do not share it with others.
+---
+## PROVISIONING
 
-## Upload the Test Data Set
+__Step 1:__ Request creation of pmi-ops.org account for Data Steward(s) by sending an email to sysadmin@pmi-ops.org with the subject   line “NEW USER REQUEST: Data Steward”. 
 
-__Note: You should not upload real participant data or de-identified data during alpha. You will be given a test data set to use instead.__
+   The email should include the following:
+   * Name and Contact (including institutional email)
+   * Organization, Role, and Justification for Access
+   * Signed copy of DRC IT Rules of Behavior for Privileged Use. Note that the entire document must be attached, not just the signature page.
```

Comment:
should there be link to this document?

Reply:
https://docs.google.com/document/d/1E6bRJ4l7AclEkaFS4Tg2zt9u3WyFMOpu4-omMjhlTRM/edit